CITATIONS.bib

```diff
@@ -0,0 +1,14 @@
+@misc{tanaka2023xiplot,
+  title         = {$\chi$iplot: Web-First Visualisation Platform for Multidimensional Data},
+  shorttitle    = {$\chi$iplot},
+  author        = {Tanaka, Akihiro and Tyree, Juniper and Bj{\"o}rklund, Anton and M{\"a}kel{\"a}, Jarmo and Puolam{\"a}ki, Kai},
+  year          = {2023},
+  month         = jun,
```

Comment:
```suggestion
  month         = {06},
```

Reply:
bibtex is smart enough to do the conversion, but why not